server/services/offendersService.js

```diff
@@ -145,31 +144,44 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
     try {
       const nomisClient = nomisClientBuilder(token)
       const uncategorisedResult = (await nomisClient.getUncategorisedOffenders(agencyId)).filter(
-        s => s.status === Status.AWAITING_APPROVAL.name
+        s => s.status === Status.AWAITING_APPROVAL.name // the status coming back from nomis
       )
 
-      if (isNilOrEmpty(uncategorisedResult)) {
+      const unapprovedWithDbRecord = await Promise.all(
+        uncategorisedResult.map(async s => {
+          const dbRecord = await formService.getCategorisationRecord(s.bookingId)
```

Comment:
Very minor but to make it consistent with line 125 you change this to:

`
uncategorisedResult.map(async s => ({
    ...s,
   ...(await formService.getCategorisationRecord(s.bookingId)),
})
`

Reply:
ok il go for it !

Reply:
think ill leave this one: the code creates a field called dbrecord and is the same as at line 111